build/validators/testnet-validator-setup.md

```diff
@@ -1,156 +1,168 @@
+	-   the  `min-self-delegation`  is the lowest amount of personal funds the validator is required to have in their own validator to stay bonded (in the example above, 1xprt)
+	-   the  `gas-prices`  is the amount of gas used to send this create-validator transaction
+
+6. **Confirm your validator is running.** Your validator is active if the following command returns anything:
+
+	```bash
+	persistenceCore query tendermint-validator-set | grep "$(persistenceCore tendermint show-address)"
+	```
+
+	You should now see your validator in [one of the Persistence explorers](https://testnet.ping.pub/test-core-1/uptime).
+7. **Congratulations!** You've just created a Persistence Testnet Validator.
+
+### View Validator Description
+
+View the validator's information with this command:
+
+```bash
+persistenceCore query staking validator <validator_address>
 ```
 
-* Install
-
-```shell
-cd persistenceCore
-make all
+Here's an example of how the above command should be used:
+```bash
+persistenceCore query staking validator persistencevaloper1rzauu3undh97yvdnj7wu2wwstm9wj8heeq2vcz
 ```
 
-### Generate keys
+### Edit Validator Description
 
-`persistenceCore keys add [key_name]`
+You can edit your validator's public description. This info is to identify your validator, and will be relied on by delegators to decide which validators to stake to. Make sure to provide input for every flag below. If a flag is not included in the command the field will default to empty (`--moniker` defaults to the machine name) if the field has never been set or remain the same if it has been set in the past.
 
-or
+The \<key\_name> specifies which validator you are editing. If you choose to not include some of the flags below, remember that the --from flag **must** be included to identify the validator to update.
 
-`persistenceCore keys add [key_name] --recover` to regenerate keys with your [BIP39](https://github.com/bitcoin/bips/tree/master/bip-0039) mnemonic
+The `--identity` can be used as to verify identity with systems like Keybase or UPort. When using Keybase, `--identity` should be populated with a 16-digit string that is generated with a [keybase.io](https://keybase.io) account. It's a cryptographically secure method of verifying your identity across multiple online networks. The Keybase API allows us to retrieve your Keybase avatar. This is how you can add a logo to your validator profile.
 
-## Validator setup
+```bash
+persistenceCore tx staking edit-validator
+  --moniker="choose a moniker" \
+  --website="https://your-website.com" \
+  --identity=6A0D65E29A4CBC8E \
+  --details="To infinity and beyond!" \
+  --chain-id=test-core-1 \
+  --gas="auto" \
+  --gas-prices="0.0025uxprt" \
+  --from=<key_name> \
+  --commission-rate="0.10"
+```
 
-### Before genesis: NOW CLOSED
+{% hint style="warning" %}
+Please note that some parameters such as `commission-max-rate` and `commission-max-change-rate` cannot be changed once your validator is up and running.
```

Comment:
`{% hint %}` does not work in github markdown 🤔 

You can use these for special messages
```
> **Note**
> This is a note

> **Warning**
> This is a warning
```
Preview
> **Note**
> This is a note

> **Warning**
> This is a warning

Reply:
Thanks for pointing this out. I'll use it going forward.

Change implemented.